samples/core/lightweight_component/lightweight_component.ipynb

```diff
@@ -29,7 +29,7 @@
    },
    "outputs": [],
    "source": [
-    "EXPERIMENT_NAME = 'lightweight python components'"
+    "experiment_name = 'lightweight python components'"
```

Comment:
what about the lightweight_component.config?

Reply:
Good catch. Removed it. I think for now we can just enforce the experiment_name to be set to test_name + -test in the sample test infra.